notifier/plotting_test.go

```diff
@@ -127,3 +130,57 @@ func TestGetMetricDataToShow(t *testing.T) {
 		So(len(metricsData), ShouldEqual, len(givenSeries))
 	})
 }
+
+func Test_fetchAvailableSeries(t *testing.T) {
```

Comment:
Naming is inconsistent with other tests in this file. `TestFetchAvailableSeries` would be better.